### PR TITLE
Use Ameba on Travis CI

### PR DESCRIPTION
**Pull request by Sija**
Because linting is caring :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ script:
   - crystal spec
   - crystal spec --release --no-debug
   - crystal tool format --check
+  - bin/ameba src
```
Comment (wontruefree) [inline .travis.yml]: Seems like the linter should go first to fail fast and not wast time on CI.

Comment (Sija) [reply .travis.yml]: Nope, I believe it's more important to see failing specs first.

Comment (wontruefree) [reply .travis.yml]: sounds good :)

**Comment by sdogruyol** [comment]
@Sija I'm not totally sure about this...yet

**Comment by Blacksmoke16** [comment]
@wisn That change was probably to get ameba to pass, due to the like `unnecessary return` assertion.  Since it was the last thing, the `return` keyword is not needed.

**Comment by wisn** [comment]
@Blacksmoke16 Ah, that makes sense to me.

File: src/kemal/ext/response.cr [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class HTTP::Server::Response
   class Output
     def close
-      unless response.wrote_headers? && !response.headers.has_key?("Content-Range")
+      if !response.wrote_headers? && response.headers.has_key?("Content-Range")
```
Comment (wontruefree) [inline src/kemal/ext/response.cr]: Is this a linting thing?

Comment (Blacksmoke16) [reply src/kemal/ext/response.cr]: Probably more so readability.  @Sija can confirm.

Comment (Sija) [reply src/kemal/ext/response.cr]: @wontruefree Yes, it's related to double negation - `unless` with `!`.

**Comment by Sija** [comment]
@sdogruyol any decisions? 🏓

**Comment by bararchy** [comment]
+1 for ameba !

**Pull request merged**